requirements.txt

```diff
@@ -0,0 +1,53 @@
+attrs==20.3.0; python_version >= "2.7" and python_full_version < "3.0.0" or python_full_version >= "3.4.0"
```

Comment:
`poetry` allows the [`export`](https://python-poetry.org/docs/cli/#export) (and import) of `requirements.txt` file, so I thought it would be nice to provide developers who want to use `requirements.txt` file with one.